### ISSUE TITLE
API requests

### ISSUE THREAD

**Issue by Kukks** [coinfoundry/miningcore#118]
- A total count should be included in the paging to be able to create a pagination component on the frontend. This currently applies to payments and blocks.
- Filter payments by miner

Out of scope for this specific task but adding an alternative api to push changes to the frontend would also be nice. Signalr core is available for ASP.NET Core 2 and is usually dead-easy to use.

**Comment by calvintam236** [comment]
I'm closing this issue because this is for poolmining.org web front-end, and not a part of the MiningCore code. Please post this request at http://poolmining.userecho.com/.

**Comment by Kukks** [comment]
No it's not. It's for the api-server inside miningcore. 
https://github.com/coinfoundry/miningcore/blob/master/src/MiningCore/Api/ApiServer.cs

I have my own custom frontend that hooks to it so I'll probably add these two in at some point tomorrow. :)

**Comment by Kukks** [comment]
@calvintam236  Can you re-open?

**Comment by calvintam236** [comment]
You'll have to wait for @oliverw since work on the first API issue #27 is not even started since October, and your second request is already listed in #27.

**Comment by oliverw** [comment]
Implemented by #758b77b. Five new API endpoints have been added:

- /api/v2/pools/`poolId`/blocks
- /api/v2/pools/`poolId`/payments
- /api/v2/pools/`poolId`/miners/`address`/payments
- /api/v2/pools/`poolId`/miners/`address`/balancechanges
- /api/v2/pools/`poolId`/miners/`address`/earnings/daily

All of which return the same information as their non-V2 counterparts with the difference that the result is enclosed in an envelope object returning paging and status information:

```json
{
  "success": true,
  "pageCount": 1,
  "result": [
  ....
  ]
}
```
